noir-projects/noir-contracts/contracts/docs/docs_example_contract/src/main.nr

```diff
@@ -1,34 +1,22 @@
 mod options;
 mod types;
```

Comment:
Since this contract is no longer used in yarn-project I deleted all the functionality that was not included in docs.